arm-recoveryservices/2015-11-10/swagger/replicationusages.json

```diff
@@ -0,0 +1,187 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2015-11-10",
+    "title": "RecoveryServicesClient",
+    "x-ms-code-generation-settings": {
+      "internalConstructors": false
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/replicationUsages": {
+      "get": {
+        "tags": [
+          "ReplicationUsages"
+        ],
+        "description": "Fetches the replication usages of the vault.",
+        "operationId": "ReplicationUsages_List",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/VaultName"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ReplicationUsageList"
+            }
+          }
+        },
+        "deprecated": false,
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ReplicationUsage": {
```

Comment:
Looks like a result-only data type. If so, please mark all properties as readonly. Same applies to `MonitoringSummary` and `JobsSummary`. If primitive member are guaranteed to exist in server responses (if they are "required" to be sent), mark them as `x-nullable: false` to provide a better user experience for the resulting .Net SDK.